core/src/test/java/com/hcl/ecomm/core/schedulers/SimpleScheduledTaskTest.java

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
@DeepaliAgrwl : why doe we have this , we are not writing schedulers .

Reply:
removed

Reply:
removed